packages/terra-data-grid/tests/wdio/flowsheet-data-grid-spec.js

```diff
@@ -0,0 +1,16 @@
+const defaultSelector = '#default-terra-flowsheet-data-grid';
+
+Terra.describeViewports('FlowsheetDataGrid', ['medium', 'large'], () => {
+  describe('Grid focus', () => {
+    beforeEach(() => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/default-flowsheet-data-grid');
+    });
+
+    it('validates that the first cell gains initial focus when Tab is pressed', () => {
```

Comment:
Isn't this test already covered by the DataGrid component?  Are we worried that we introduced logic that requires us to retest this functionality?

Reply:
Correct, I can probably change this to simply display the component rather than testing the tab stop. That should be covered by the DataGrid test